txpipe/random_cats.py

```diff
@@ -8,8 +8,8 @@ class TXRandomCat(PipelineStage):
     name='TXRandomCat'
     inputs = [
         ('diagnostic_maps', DiagnosticMaps),
-        ('tomography_catalog', TomographyCatalog),
-        ('photoz_stack', HDFFile),
+        ('shear_tomography_catalog', TomographyCatalog),  #WHY?
```

Comment:
We give the random objects ellipticities too, hence using that catalog.  But it can just use tracer_metadata instead, since it just needs sigma_e.  Could you change this?

Reply:
done